### DISCUSSION TITLE
Carla imports RoadRunner custom maps, some material not shown

### DISCUSSION THREAD

**Discussion by lanlandawang** [carla-simulator/carla#7070] [Q&A]
I added my own modeled material in RoadRunner and used this material in the map, imported it into Carla and this material doesn't show up!

**Comment by Jinviz** [comment]
Didn't you just move it into a folder? 
It would be nice to check if the import process went through the unreal engine normally.
Or does that mean some elements are missing from the map?

**Reply by lanlandawang** [reply]
I don't quite understand what you mean, I use the make import command to import, but it doesn't show my customized material, can you elaborate?

**Reply by Jinviz** [reply]
Is the modeled data a traffic light or another mesh?

**Reply by CFZ3668** [reply]
Yes, I have encountered a situation where traffic lights and trees cannot be displayed properly. How can I solve this problem？Looking forward to your reply！

**Comment by romanmatus** [comment]
Hello @lanlandawang did you manage to fix this issue ? If so, could you please share the map ?
Because iam searching for the map with for example country lates on sand, but not sure if this is even possible. 
Thank you.
